docs/build-instructions/freebsd.md

```diff
@@ -12,13 +12,31 @@ FreeBSD -RELEASE 64-bit is the recommended platform.
 
 ## Instructions
 
+If you have problems with permissions don't forget to prefix with `sudo`
+
+1. Clone the Atom repository:
+
   ```sh
   git clone https://github.com/atom/atom
   cd atom
-  script/build # Creates application at $TMPDIR/atom-build/Atom
-  sudo script/grunt install # Installs command to /usr/local/bin/atom
   ```
 
+2. Build Atom:
+
+  ```sh
+  script/build
+  ```
+
+  This will create the atom application at `$TMPDIR/atom-build/Atom`.
+
+4. Install the `atom` command to `/usr/local/bin/atom` by executing:
```

Comment:
I think this is step 3? (FWIW, it's usually better to not try to manually number list items, and instead denote everything with `1.` because the Markdown renderer will number things for you.)